### ISSUE TITLE
change in tax brackets in app_reform

### ISSUE THREAD

**Issue by shikharpant** [TPRU-demo/pitaxcalc-demo#55]
If the app_reform as set as below, it throws errors.

```
{
    "policy": {
      "_tbrk2": {"2017": [700000]}
    }
}

```

Output is as below

Tax-Calculator startup automatically extrapolated your data to 2017.
`Traceback (most recent call last):
  File "app17.py", line 18, in <module>
    pol.implement_reform(reform['policy'])
  File "C:\Users\pc\Desktop\Root\Github\pitaxcalc-demo\taxcalc\policy.py", line 215, in implement_reform
    self._validate_parameter_values(reform_parameters)
  File "C:\Users\pc\Desktop\Root\Github\pitaxcalc-demo\taxcalc\policy.py", line 421, in _validate_parameter_values
    assert pvalue.shape == vvalue.shape
AssertionError`

**Comment by martinholmer** [comment]
See pull request #60 for a proposed resolution for issues #55 and #56.

@indira-tpru @adc2-tpru @keshavchoudhary87 @sebastiansajie

**Comment by martinholmer** [comment]
Merge of pull request #60 resolves issue #55.
